### PR TITLE
Added IPv6 support

### PR DESCRIPTION
**Pull request by noahlvb**
Added support for IPv6 according to issue #11

### CODE REVIEW

File: ipfromwebpage/ipfromwebpage.py [hunk 1]
```diff
@@ -90,15 +90,43 @@ def ip_from_string(string):
...
     webpage_text = get_webpage_text(url)
     address_list = ip_from_string(webpage_text)
+    addressv6_list = ipv6_from_string(webpage_text)
     if address_list:
+        print('IPv4 addresses:')
         for cidr in address_list.iter_cidrs():
             print(cidr)
     else:
-        print("No ips found when scraping {}".format(url))
+        print("No ipv4s found when scraping {}".format(url))
```
Comment (shepherdjay) [inline ipfromwebpage/ipfromwebpage.py]: Looks like this line is causing some of the tests to fail as the output changed.

File: ipfromwebpage/ipfromwebpage.py [hunk 2]
```diff
@@ -90,15 +90,43 @@ def ip_from_string(string):
...
 
+    if addressv6_list:
+        print('\nIPv6 addresses:')
+        for cidr in addressv6_list.iter_cidrs():
+            print(cidr)
+    else:
+        print("No ipv6s found when scraping {}".format(url))
```
Comment (shepherdjay) [inline ipfromwebpage/ipfromwebpage.py]: This line as well causes some of the tests to fail. Those tests will need to be adjusted.

File: ipfromwebpage/ipfromwebpage.py [hunk 3]
```diff
@@ -90,15 +90,43 @@ def ip_from_string(string):
...
+    """
+
+    ipv6_regex = re.compile('((([0-9A-Fa-f]{1,4}:){7}([0-9A-Fa-f]{1,4}|:))|(([0-9A-Fa-f]{1,4}:){6}(:[0-9A-Fa-f]{1,4}|((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{1,4}){1,2})|:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){4}(((:[0-9A-Fa-f]{1,4}){1,3})|((:[0-9A-Fa-f]{1,4})?:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){1,4})|((:[0-9A-Fa-f]{1,4}){0,2}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,4}){1,5})|((:[0-9A-Fa-f]{1,4}){0,3}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){1,6})|((:[0-9A-Fa-f]{1,4}){0,4}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(:(((:[0-9A-Fa-f]{1,4}){1,7})|((:[0-9A-Fa-f]{1,4}){0,5}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:)))(%.+)?(\/[0-9][0-9]?|1([01][0-9]|2[0-8]))?')
```
Comment (shepherdjay) [inline ipfromwebpage/ipfromwebpage.py]: Regex looks good on first glance but we need to add some tests as well to test this new functionality against some valid and invalid IP addresses.

**Comment by shepherdjay** [comment]
@noahlvb let me know if you need any assistance with the tests on this, at a quick glance your implementation looks solid.

**Comment by noahlvb** [comment]
Some assistance would be great since I have never really made tests my self for python.

**Comment by shepherdjay** [comment]
Hey looks like you got the tests adjusted appropriately. I'm going to go ahead and pull these in. Not sure why codecov is throwing a partial

**Pull request merged**